src/components/select/select.tsx

```diff
@@ -57,6 +57,11 @@ export class Select implements FormComponent {
    */
   @Prop() readonly readonly: boolean;
 
+  /**
+   * Enable a text imput with suggest values.
```

Comment:
```suggestion
   * Render a text input showing a list of suggested elements.
```

Reply:
Ready! Check the commit 15714ff to see the change